lib/Conversion/Utils/Utils.cpp

```diff
@@ -275,6 +275,8 @@ Value convertScalarToDtype(OpBuilder &b, Location loc, Value scalar,
   llvm_unreachable("convertScalarToDtype should handle all the types");
 }
 
+bool isBuiltInType(Type type) { return isa<BuiltinDialect>(type.getDialect()); }
```

Comment:
Should this be in `lib/Dialect/Torch/Utils/Utils.cpp` since its prototype is in `include/torch-mlir/Dialect/Torch/Utils/Utils.h`?

Reply:
yea!